torchvision/prototype/transforms/_utils.py

```diff
@@ -98,6 +98,22 @@ def query_chw(sample: Any) -> Tuple[int, int, int]:
     return c, h, w
 
 
+def query_hw(sample: Any) -> Tuple[int, int]:
+    flat_sample, _ = tree_flatten(sample)
+    hws = {
+        tuple(get_spatial_size(item))
+        for item in flat_sample
+        if isinstance(item, (features.Image, PIL.Image.Image, features.Video, features.Mask, features.BoundingBox))
+        or features.is_simple_tensor(item)
+    }
+    if not hws:
+        raise TypeError("No image, video, mask or bounding box was found in the sample")
+    elif len(hws) > 1:
+        raise ValueError(f"Found multiple HxW dimensions in the sample: {sequence_to_str(sorted(hws))}")
+    h, w = hws.pop()
+    return h, w
```

Comment:
Lots of code duplication with `query_chw`. The two methods differ on the callable, the checked types, the error messages and the return type. I was tempted to write something that passes a callable and tries to reduce duplicate code but it become unnecessarily complex. Happy to implement other approaches if you have better ideas.